tests/integration_tests.rs

```diff
@@ -7,10 +7,10 @@ mod common;
 fn writing_a_key_should_return_same_value() {
     let mut db = common::DatabaseTesting::new("db1".to_owned(), ByteSize::b(1).as_u64());
 
-    db.write("name".as_bytes(), "Peter".as_bytes()).unwrap();
```

Comment:
`&'static str::as_bytes()` can be a bytestring literal (`b"foobar"`) instead.

up until very recently, `str::as_bytes()` wasn't const so there could be some overhead introduced here.